### PR TITLE
initial pipeline test yaml #1761

### PR DESCRIPTION
**Pull request by ar3ndt**

### CODE REVIEW

**Review by atsikham — CHANGES_REQUESTED** [review]
A few points to discuss/change:
* Where is test results publishing? Or it's not generated in this module?
* Please use `displayName` for jobs and use more descriptive job names, as we discussed.
* Do not forget disable old pipelines in UI.

**Comment by ar3ndt** [comment]
> A few points to discuss/change:
> 
> * Where is test results publishing? Or it's not generated in this module?
> * Please use `displayName` for jobs and use more descriptive job names, as we discussed.
> * Do not forget disable old pipelines in UI.

reports are not generated for now afaik

**Review by atsikham — APPROVED** [review]
For me changes are ok. Contact @to-bar if job names are ok for him.

**Review by mkyc — CHANGES_REQUESTED** [review]
After PR #25 this one should definitely look [more like this](https://github.com/epiphany-platform/m-azure-basic-infrastructure/blob/develop/pipeline.yml)

File: pipeline.yml [hunk 1]
```diff
@@ -1,12 +1,106 @@
...
+            echo "##vso[task.setvariable variable=PATH]$(echo $GOPATH:$GOBIN:$PATH)"
+      - task: Bash@3
+        displayName: 'Run Tests'
+        inputs:
+          targetType: 'inline'
+          script: |
+            set -eu
+            echo "Start make test"
+            export AWS_ACCESS_KEY_ID=$(AWS_ACCESS_KEY_ID)
+            export AWS_SECRET_ACCESS_KEY=$(AWS_SECRET_ACCESS_KEY)
```
Comment (atsikham) [inline pipeline.yml:59]: Is it possible to replace env related lines in [this](https://github.com/epiphany-platform/m-azure-basic-infrastructure/blob/a72f72aea681483ef3bf56873246a0d3e4993393/pipeline.yml#L36) way?

**Review by atsikham — CHANGES_REQUESTED** [review]

**Comment by mkyc** [comment]
Move source branch to epiphany repo. Otherwise Azure DevOps would not be able to run tests correctly.

**Review by atsikham — APPROVED** [review]

**Comment by mkyc** [comment]
> Move source branch to epiphany repo. Otherwise Azure DevOps would not be able to run tests correctly.

Haven't seen it reported even once. And reason is wrong source.

**Comment by ar3ndt** [comment]
moved to source repo

**Pull request closed without merging**